### ISSUE TITLE
learning rate update

### ISSUE THREAD

**Issue by slala2121** [MiaoyunZhao/GANTransferLimitedData#3]
Hi,

Minor issue but seems like the lr scheduler step should be called after the training and discriminator training steps (pytorch 1.1.0+) else the first lr is skipped.
